File: grouping-client/src/app/models/review.ts
import { Site } from "./site";

export class Review {

  constructor(
    private _id: number,
    private _groupId: number,
    private _site: Site,
    private _userId: number,
    private _rating: number,
    private _comment: string,
    private _tocken: string | null
  ) {}

  public get tocken(): string | null {
    return this._tocken;
  }
  public set tocken(value: string | null) {
    this._tocken = value;
  }

  public get comment(): string {
    return this._comment;
  }
  public set comment(value: string) {
    this._comment = value;
  }
  public get rating(): number {
    return this._rating;
  }
  public set rating(value: number) {
    this._rating = value;
  }
  public get userId(): number {
    return this._userId;
  }
  public set userId(value: number) {
    this._userId = value;
  }
  public get site(): Site {
    return this._site;
  }
  public set site(value: Site) {
    this._site = value;
  }
  public get groupId(): number {
    return this._groupId;
  }
  public set groupId(value: number) {
    this._groupId = value;
  }
  public get id(): number {
    return this._id;
  }
  public set id(value: number) {
    this._id = value;
  }
}

File: sites-service/src/main/java/com/ironhack/sitesservice/model/Review.java
package com.ironhack.sitesservice.model;

import javax.persistence.*;

@Entity
public class Review {

    /**
     * Properties
     */
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private Long groupId;
    @ManyToOne
    private Site site;
    private Long userId;
    private Integer rating;
    private String comment;

    /**
     * Default constructor
     */
    public Review() {
    }

    /**
     * Constructor: groupId, site, userId, rating, comment
     */
    public Review(Long groupId, Site site, Long userId, Integer rating, String comment) {
        this.groupId = groupId;
        this.site = site;
        this.userId = userId;
        this.rating = rating;
        this.comment = comment;
    }

    /**
     * Getters & Setters
     */
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getGroupId() {
        return groupId;
    }

    public void setGroupId(Long groupId) {
        this.groupId = groupId;
    }

    public Site getSite() {
        return site;
    }

    public void setSite(Site site) {
        this.site = site;
    }

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    public Integer getRating() {
        return rating;
    }

    public void setRating(Integer rating) {
        this.rating = rating;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

    @Override
    public String toString() {
        return "Review{" +
                "id=" + id +
                ", groupId=" + groupId +
                ", site=" + site +
                ", userId=" + userId +
                ", rating=" + rating +
                ", comment='" + comment + '\'' +
                '}';
    }
}